### ISSUE TITLE
Filtering of Parameters in Rails 4

### ISSUE THREAD

**Issue by benarent** [airbrake/airbrake#345]
As reported via support. 

https://github.com/airbrake/airbrake/blob/master/lib/airbrake/rails/controller_methods.rb#L42-L61 
airbrake should be using the rails3 codepath on rails 4

**Comment by joevandyk** [comment]
is this fixed?

**Comment by djhopper01** [comment]
This is fixed in master; however, the latest release does not include the fix.

v4.1.0 from Rubygems: parameters in `config.filter_parameters` are not filtered.
latest from master branch: parameters in `config.filter_parameters` are filtered correctly.

Also, the fix for this seems to be in this commit: https://github.com/airbrake/airbrake/commit/476e426643d94bd4f6c4d2b55019828d148d85e9 on September 23, 2014. v4.1.0 is the latest release and was released on September 4, 2014.

I simply started tracking master and parameters were filtered as they should.
